### PR TITLE
#261 fix default time for edit blue delta

### PR DESCRIPTION
**Pull request by smebrahimi**
This issue #261

### CODE REVIEW

File: src/components/BaseTimePicker.vue [hunk 1]
```diff
@@ -77,7 +72,8 @@ export default {
     }
   },
   watch: {
-    selectedTime(value) {
+    defaultTime(value) {
```
Comment (mort3za) [inline src/components/BaseTimePicker.vue:75]: I guess we need `immediate:true` here for some situations.

File: src/views/BlueDeltaSettings.vue [hunk 2]
```diff
@@ -195,6 +195,10 @@ export default {
         time.push(i);
       }
       return time;
+    },
+    dateTimezon() {
```
Comment (atharva3010) [inline src/views/BlueDeltaSettings.vue:199]: Just a small typo, it should be `dateTimezone` instead of `dateTimezon`

**Comment by paulchrisluke** [comment]
Ux is working! https://media.tenor.com/images/848f027efe1d7c4978affa53efe54567/tenor.gif

**Comment by mort3za** [comment]
@atharva3010 we don't need branches after merge

**Pull request merged**